### PR TITLE
Dịch "linear-regression-gluon_vn" - Phần 1

### LINKED ISSUE #664: Dịch "linear-regression-gluon_vn" - Phần 1
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_linear-networks/linear-regression-gluon_vn.md#L1-L48

### PR DESCRIPTION
**Pull request by tamnguyenvan**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #664

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm [hướng dẫn hoặc yêu cầu dịch hình](../img/README.md).

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [x] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](../docs/contributors_info.md/)**.

## Với reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 1]
```diff
@@ -15,25 +15,27 @@ In the previous section, we relied only on (i) `ndarray` for data storage and li
 In practice, because data iterators, loss functions, optimizers, and neural network layers (and some whole architectures) are so common, modern libraries implement these components for us as well.
 -->
 
-*dịch đoạn phía trên*
+Sự quan tâm nhiệt thành và rộng khắp tới học sâu một vài năm gần đây đã tạo cảm hứng cho cả các công ty, học viện và những người đam mê phát triển nhiều framework nguồn mở mạnh mẽ, giúp tự động hóa các công việc lặp đi lặp lại trong quá trình triển khai các thuật toán học dựa trên gradient.
```
Comment (rootonchair) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Sự quan tâm nhiệt thành và rộng khắp tới học sâu một vài năm gần đây đã tạo cảm hứng cho cả các công ty, học viện và những người đam mê phát triển nhiều framework mã nguồn mở mạnh mẽ, giúp tự động hóa các công việc lặp đi lặp lại trong quá trình triển khai các thuật toán học dựa trên gradient.
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 2]
```diff
@@ -15,25 +15,27 @@ In the previous section, we relied only on (i) `ndarray` for data storage and li
...
+Trong chương trước, chúng ta chỉ dựa trên (i) `ndarray` để lưu dữ liệu và thực hiện tính toán đại số tuyến tính; và (ii) `autograd` để thực hiện tính đạo hàm.
```
Comment (rootonchair) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Trong chương trước, chúng ta chỉ dựa vào (i) `ndarray` để lưu dữ liệu và thực hiện tính toán đại số tuyến tính; và (ii) `autograd` để thực hiện tính đạo hàm.
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 3]
```diff
@@ -15,25 +15,27 @@ In the previous section, we relied only on (i) `ndarray` for data storage and li
...
+Trong thực hành, do các bộ duyệt dữ liệu, các hàm mất mát, các bộ tối ưu, và các tầng của mạng nơ-ron (thậm chí là toàn bộ kiến trúc) rất phổ biển, các thư viện hiện đại sẽ triển khai sẵn những thành phần này cho chúng ta.
```
Comment (rootonchair) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Trong thực tế, do các bộ duyệt dữ liệu, các hàm mất mát, các bộ tối ưu, và các tầng của mạng nơ-ron (thậm chí là toàn bộ kiến trúc) rất phổ biển, các thư viện hiện đại sẽ triển khai sẵn những thành phần này cho chúng ta.
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 4]
```diff
@@ -15,25 +15,27 @@ In the previous section, we relied only on (i) `ndarray` for data storage and li
...
 
 <!--
 In this section, we will show you how to implement the linear regression model from :numref:`sec_linear_scratch` concisely by using Gluon.
 -->
 
-*dịch đoạn phía trên*
+Trong chương này, chúng tôi sẽ hướng dẫn bạn làm cách nào để triển khai mô hình hồi quy tuyến tính trong phần :numref:`sec_linear_scratch` một cách chi tiết sử dụng Gluon.
```
Comment (rootonchair) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Mục này sẽ hướng dẫn bạn làm cách nào để triển khai mô hình hồi quy tuyến tính trong phần :numref:`sec_linear_scratch` một cách chi tiết với Gluon.
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 5]
```diff
@@ -15,25 +15,27 @@ In the previous section, we relied only on (i) `ndarray` for data storage and li
...
 
-## *dịch tiêu đề phía trên*
+## Tạo Tập dữ liệu
 
 <!--
 To start, we will generate the same dataset as in the previous section.
 -->
 
-*dịch đoạn phía trên*
+Để bắt đầu, chúng ta sẽ sinh tập dữ liệu giống như chương trước.
```
Comment (rootonchair) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Để bắt đầu, chúng ta sẽ tạo một tập dữ liệu giống như mục trước.
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 6]
```diff
@@ -15,25 +15,27 @@ In the previous section, we relied only on (i) `ndarray` for data storage and li
 In practice, because data iterators, loss functions, optimizers, and neural network layers (and some whole architectures) are so common, modern libraries implement these components for us as well.
 -->
 
-*dịch đoạn phía trên*
+Sự quan tâm nhiệt thành và rộng khắp tới học sâu một vài năm gần đây đã tạo cảm hứng cho cả các công ty, học viện và những người đam mê phát triển nhiều framework mã nguồn mở mạnh mẽ, giúp tự động hóa các công việc lặp đi lặp lại trong quá trình triển khai các thuật toán học dựa trên gradient.
```
Comment (tiepvupsu) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Sự quan tâm nhiệt thành và rộng khắp tới học sâu một vài năm gần đây đã tạo cảm hứng cho cả các công ty, học viện và những người đam mê phát triển nhiều framework mã nguồn mở hoàn thiện, giúp tự động hóa các công việc lặp đi lặp lại trong quá trình triển khai các thuật toán học dựa trên gradient.
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 7]
```diff
@@ -15,25 +15,27 @@ In the previous section, we relied only on (i) `ndarray` for data storage and li
...
+Trong chương trước, chúng ta chỉ dựa vào (i) `ndarray` để lưu dữ liệu và thực hiện tính toán đại số tuyến tính; và (ii) `autograd` để thực hiện tính đạo hàm.
+Trong thực tế, do các bộ duyệt dữ liệu, các hàm mất mát, các bộ tối ưu, và các tầng của mạng nơ-ron (thậm chí là toàn bộ kiến trúc) rất phổ biển, các thư viện hiện đại sẽ triển khai sẵn những thành phần này cho chúng ta.
```
Comment (tiepvupsu) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Trong thực tế, do các bộ duyệt dữ liệu, các hàm mất mát, các bộ tối ưu, và các tầng của mạng nơ-ron (thậm chí là toàn bộ kiến trúc) rất phổ biển, các thư viện hiện đại đã triển khai sẵn những thành phần này cho chúng ta.
```

`iterator` có lẽ được dùng nhiều trong khoa học máy tính rồi. Bạn nào biết từ này được dịch là gì trong các giáo trình không? Mình không nghĩ nó là `bộ duyệt`.

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 8]
```diff
@@ -15,25 +15,27 @@ In the previous section, we relied only on (i) `ndarray` for data storage and li
 In practice, because data iterators, loss functions, optimizers, and neural network layers (and some whole architectures) are so common, modern libraries implement these components for us as well.
 -->
 
-*dịch đoạn phía trên*
+Sự quan tâm nhiệt thành và rộng khắp tới học sâu một vài năm gần đây đã tạo cảm hứng cho cả các công ty, học viện và những người đam mê phát triển nhiều framework mã nguồn mở hoàn thiện, giúp tự động hóa các công việc lặp đi lặp lại trong quá trình triển khai các thuật toán học dựa trên gradient.
```
Comment (tanduong) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Sự quan tâm nhiệt thành và rộng khắp tới học sâu trong những năm gần đây đã tạo cảm hứng cho các công ty, học viện và những người đam mê phát triển nhiều framework nguồn mở tới giai đoạn hoàn thiện, giúp tự động hóa các công việc lặp đi lặp lại trong quá trình triển khai các thuật toán học dựa trên gradient.
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 9]
```diff
@@ -15,25 +15,27 @@ In the previous section, we relied only on (i) `ndarray` for data storage and li
...
 
-## *dịch tiêu đề phía trên*
+## Tạo Tập dữ liệu
 
 <!--
 To start, we will generate the same dataset as in the previous section.
 -->
 
-*dịch đoạn phía trên*
+Để bắt đầu, chúng ta sẽ tạo một tập dữ liệu giống như mục trước.
```
Comment (tanduong) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Để bắt đầu, chúng ta hãy tạo một tập dữ liệu như mục trước.
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 10]
```diff
@@ -15,25 +15,27 @@ In the previous section, we relied only on (i) `ndarray` for data storage and li
...
+Sự quan tâm nhiệt thành và rộng khắp tới học sâu một vài năm gần đây đã tạo cảm hứng cho cả các công ty, học viện và những người đam mê phát triển nhiều framework mã nguồn mở hoàn thiện, giúp tự động hóa các công việc lặp đi lặp lại trong quá trình triển khai các thuật toán học dựa trên gradient.
+Trong chương trước, chúng ta chỉ dựa vào (i) `ndarray` để lưu dữ liệu và thực hiện tính toán đại số tuyến tính; và (ii) `autograd` để thực hiện tính đạo hàm.
+Trong thực tế, do các bộ duyệt dữ liệu, các hàm mất mát, các bộ tối ưu, và các tầng của mạng nơ-ron (thậm chí là toàn bộ kiến trúc) rất phổ biển, các thư viện hiện đại đã triển khai sẵn những thành phần này cho chúng ta.
 
 <!--
 In this section, we will show you how to implement the linear regression model from :numref:`sec_linear_scratch` concisely by using Gluon.
 -->
 
-*dịch đoạn phía trên*
+Mục này sẽ hướng dẫn bạn làm cách nào để triển khai mô hình hồi quy tuyến tính trong phần :numref:`sec_linear_scratch` một cách chi tiết với Gluon.
```
Comment (tanduong) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Mục này sẽ hướng dẫn bạn làm cách nào để xây dựng mô hình hồi quy tuyến tính trong phần :numref:`sec_linear_scratch` một cách xúc tích với Gluon.
```

Comment (tamnguyenvan) [reply chapter_linear-networks/linear-regression-gluon_vn.md]: `concisely` là chính xác mà bác nhỉ

Comment (tiepvupsu) [reply chapter_linear-networks/linear-regression-gluon_vn.md]: `precise` là `chính xác`, `concise` là `ngắn gọn`, `xúc tích`.

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 11]
```diff
@@ -5,7 +5,7 @@
 # Concise Implementation of Linear Regression
 -->
 
-# *dịch tiêu đề phía trên*
+# Triển khai chi tiết Hồi quy tuyến tính
```
Comment (tiepvupsu) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
# Triển khai xúc tích của Hồi quy Tuyến tính
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 12]
```diff
@@ -15,25 +15,27 @@ In the previous section, we relied only on (i) `ndarray` for data storage and li
 In practice, because data iterators, loss functions, optimizers, and neural network layers (and some whole architectures) are so common, modern libraries implement these components for us as well.
 -->
 
-*dịch đoạn phía trên*
+Sự quan tâm nhiệt thành và rộng khắp tới học sâu trong những năm gần đây đã tạo cảm hứng cho các công ty, học viện và những người đam mê phát triển nhiều framework nguồn mở tới giai đoạn hoàn thiện, giúp tự động hóa các công việc lặp đi lặp lại trong quá trình triển khai các thuật toán học dựa trên gradient.
+Trong chương trước, chúng ta chỉ dựa vào (i) `ndarray` để lưu dữ liệu và thực hiện tính toán đại số tuyến tính; và (ii) `autograd` để thực hiện tính đạo hàm.
+Trong thực tế, do các bộ duyệt dữ liệu, các hàm mất mát, các bộ tối ưu, và các tầng của mạng nơ-ron (thậm chí là toàn bộ kiến trúc) rất phổ biển, các thư viện hiện đại đã triển khai sẵn những thành phần này cho chúng ta.
```
Comment (tiepvupsu) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Trên thực tế, do các bộ duyệt dữ liệu, các hàm mất mát, các bộ tối ưu, và các tầng của mạng nơ-ron (thậm chí là toàn bộ kiến trúc) rất phổ biển, các thư viện hiện đại đã triển khai sẵn những thành phần này cho chúng ta.
```
